helm-charts/aerospike-cluster/values.yaml

```diff
@@ -68,7 +68,7 @@ podSpec: {}
 #   multiPodPerHost: true
 #    sidecars:
```

Comment:
Not related to your tag changes but since we are fixing this side-car section, can you please also fix the indentation of `sidecars`. Should be at the same level as multiPodPerHost

```
   multiPodPerHost: true
   sidecars:
     - name: aerospike-prometheus-exporter
       image: aerospike/aerospike-prometheus-exporter:v1.18.0
       ports:
         - containerPort: 9145
           name: exporter
```